README.md

```diff
@@ -35,6 +35,24 @@ To specify the formatter in XML `<appSettings>` provide its assembly-qualified t
   <add key="serilog:write-to:File.formatter"
        value="Serilog.Formatting.Compact.CompactJsonFormatter, Serilog.Formatting.Compact" />
 ```
+#### JSON `appsettings.json` configuration
+To specify formatter in json `appsettings.json` provide its assembly-qualified type name:
+
+```json
+{
+  "Serilog": {
+    "WriteTo": [
+      {
+        "Name": "RollingFile",
```

Comment:
`RollingFile` is being deprecated in favour of `File` - it would be nice to show that here (the `pathFormat` argument becomes `path`). I notice now that there's still a mention of `RollingFile` in the XML example a few lines up; it would be nice to fix that too, if possible.

Reply:
Sure. Updated.